extensions/amp-copy/amp-copy.md

```diff
@@ -0,0 +1,132 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-copy"></a> `amp-copy`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Shows text, with a button to copy to the user's clipboard.</td>
+  </tr>
+  <tr>
+    <!-- TODO: Change this? -->
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
```

Comment:
oh and you will need to create an experiment and guard the build callback with that expt.